status-panels-lib/status-panels/main.libsonnet

```diff
@@ -0,0 +1,40 @@
+local panels = import './panels.libsonnet';
+local rows = import './rows.libsonnet';
+local targets = import './targets.libsonnet';
+local variables = import './variables.libsonnet';
+local g = import 'github.com/grafana/grafonnet/gen/grafonnet-latest/main.libsonnet';
```

Comment:
There was a suggestion from as code team to move this dependency in g.libsonnet file and then reference that file. So If you want to change lib version - you change it one place.
https://github.com/grafana/jsonnet-libs/pull/1028/files#diff-cf98599fcd312ea3c0b63c5a9a97a9f4e2ae2cd6c511952c956aa2ab1d703ba3

Reply:
Done